Cesium.CodeGen/Ir/Expressions/Constants/CharConstant.cs

```diff
@@ -41,10 +41,11 @@ private static char UnescapeCharacter(string text)
             'n' => '\n',
             'r' => '\r',
             't' => '\t',
-            'v' => '\v',
+            '?' => '?',
+            'v' => (char)0x0b,
```

Comment:
Huh, is there a reason you've decided to replace `'\v'` with `0x0b`? As far as I see, in C# it's the same thing, so it should work as it was 😅